lib/utils/jsonPathUtils.ts

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import jp from 'jsonpath';
 
-import { InputFieldType } from '../verifiablePresentation/SSI.types';
+import { InputFieldType } from '../types/SSI.types';
 
 export class JsonPathUtils {
   /**
-   * @param verifiableCredential: a vc object can be found in verifiablePresentation.verifiableCredential[i]
+   * @param verifiableCredential: a vc object can be found in types.verifiableCredential[i]
```

Comment:
unexpected edit, I think.
we can revert this.

Reply:
fixed